AUTH-README.md

```diff
@@ -0,0 +1,59 @@
+# Authentication
+
+As outlined <a href="https://github.com/grpc/grpc-common/blob/master/grpc-auth-support.md">here</a> gRPC supports a number of different mechanisms for asserting identity between an client and server. We'll present some code-samples here demonstrating how to provide SSL/TLS support encryption and identity assertions as well as passing OAuth2 tokens to services that support it.
```

Comment:
Would suggest complete rephrase (waves technical writer sword of fire):

As outlined in <a href="https://github.com/grpc/grpc-common/blob/master/grpc-auth-support.md">gRPC Authentication Support</a>, gRPC supports a number of different mechanisms for asserting identity between a client and server. This document provides code samples demonstrating how to provide SSL/TLS support encryption and identity assertions in Java, as well as passing OAuth2 tokens to services that support it.